packages/fluentui/react-telemetry/src/TelemetryPopover.tsx

```diff
@@ -0,0 +1,117 @@
+import { useEventListener } from '@fluentui/react-component-event-listener';
+import { StylesContextInputValue, Telemetry } from '@fluentui/react-bindings';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { ThemeProvider } from 'react-fela';
+
+import * as styles from './styles';
+import { TelemetryTable } from './TelemetryTable';
+import { useTelemetryState } from './useTelemetryState';
+import { TelemetryPerfFlags } from './TelemetryPerfFlags';
+
+export type TelemetryPopoverProps = {
+  mountNode?: HTMLElement;
+  hotKeyHandler?: (e: KeyboardEvent) => boolean;
+};
+
+function defaultHotKeyHandler(e: KeyboardEvent) {
+  return e.altKey && e.shiftKey && e.code === 'KeyT';
+}
+
+export const TelemetryPopover: React.FC<TelemetryPopoverProps> = props => {
+  const { children, hotKeyHandler = defaultHotKeyHandler, mountNode = document.body } = props;
+
+  const telemetry = React.useMemo(() => new Telemetry(), []);
+  const [state, dispatch] = useTelemetryState();
+
+  useEventListener({
+    listener: e => {
+      if (state.visible) {
+        return;
+      }
+
+      if (hotKeyHandler(e)) {
+        dispatch({ type: 'SET_VISIBILITY', value: true });
+      }
+    },
+    target: mountNode.ownerDocument || mountNode,
+    type: 'keydown',
+  });
+
+  const outgoingContext = React.useMemo<Partial<StylesContextInputValue>>(
+    () => ({
+      performance: state.performanceFlags,
+      telemetry: telemetry,
+    }),
+    [telemetry, state.performanceFlags],
+  );
+
+  return (
+    <>
+      <ThemeProvider theme={outgoingContext} overwrite>
```

Comment:
The same comment as for the `TelemetryTable` - This is a nice way to set telemetry for the `Provider`. How should a consumer do that without using `TelemetryPopover`?